### PR TITLE
DOC-7538: Mark channels as favorites page needs animated GIFs for mobile app workflows

### LINKED ISSUE #7538: Mark channels as favorites page needs animated GIFs for mobile app workflows
## Context

To help visual learners learn how to perform tasks using the Mattermost mobile app, add visuals and short animated GIFs on the [Mark channels as favorites](https://docs.mattermost.com/collaborate/favorite-channels.html) page.

## Docs Request

- Add appropriate visuals/GIFs for all the steps in the `Mark channels as favorite` section under the mobile app tab.

## Additional Notes

- Use the latest Mattermost mobile app and the standard Mattermost Denim theme to capture all workflows.
- Avoid drawing lines, arrows, boxes, or callouts on images.
- Don't forget to include ALT Text descriptions for all new image assets.
- Use existing docs pages with images as a starting point for code guidance. The Docs team can help provide RST syntax support for publication.
- Save static image files in PNG format within the docs repository's `/images` directory.

# QA Validation Criteria

In a generated docs preview:
- Visiting the docs page results in appropriate new mobile-specific screenshots in each subsection from the **Mobile** tab.

### PR DESCRIPTION
**Pull request by Rutam21**
### Summary

This PR adds visual workflows for the Mattermost mobile app tab on the [Mark channels as favorites](https://docs.mattermost.com/collaborate/favorite-channels.html) page.

### Ticket Link

Fixes #7538 

- [x] This PR follows Mattermost Contributing Guidelines.
- [x] The changes are tested in a local docs build.

### CODE REVIEW

**Comment by Rutam21** [comment]
@cwarnermm Please review and suggest changes, if any. Thanks.

@lindy65 Please add this to my queue for QA Testing, if needed. Thank you.

**Comment by lindy65** [comment]
Assigned to you for testing @Rutam21 👍

**Comment by Rutam21** [comment]
## The changes are tested in the preview environment and reflect properly.

- The `Mark channels as favorites` page now has respective workflow images under the mobile tab for visual learners.

Please take a look at the screenshots below for confirmation.

## Current Docs Page [[Mark channels as favorites](https://docs.mattermost.com/collaborate/favorite-channels.html)]

Screenshot 2024-11-05 032111

## Preview Environment Page Video

https://github.com/user-attachments/assets/cb97d61d-cca3-45f5-9966-6a61697bd236


**[ Awaiting final review from @cwarnermm ]**

cc: @lindy65

**Comment by lindy65** [comment]
Thanks for testing @Rutam21 :)

File: source/collaborate/favorite-channels.rst [hunk 1]
```diff
@@ -30,7 +30,45 @@ To mark a channel as a **Favorite**:
...
+      :alt: Tap on More options to access available options for the channel.
+      :scale: 30
+
   3. Tap **Favorite**.
+
+  .. image:: ../images/mobile-select-view-info-for-a-channel.jpg
+      :alt: Tap on Favorite to mark the channel as one of the favorites.
+      :scale: 30
+
+.. note::
```
Comment (cwarnermm) [inline source/collaborate/favorite-channels.rst:50]:
```suggestion
.. tip::
```

File: source/collaborate/favorite-channels.rst [hunk 2]
```diff
@@ -30,7 +30,45 @@ To mark a channel as a **Favorite**:
...
+
   3. Tap **Favorite**.
+
+  .. image:: ../images/mobile-select-view-info-for-a-channel.jpg
+      :alt: Tap on Favorite to mark the channel as one of the favorites.
+      :scale: 30
+
+.. tip::
+
+  You can also mark a channel as favorite in this way.
```
Comment (cwarnermm) [inline source/collaborate/favorite-channels.rst:52]:
```suggestion
  Alternatively, you can mark a favorite channel as follows:
```

**Pull request merged**